hztb-servicemanager/src/main/java/com/svs/hztb/service/impl/GCMServiceImpl.java

```diff
@@ -99,7 +99,7 @@ public void sendWelcomeNotification(RequestData requestData, User user) {
 		try {
 			stepDefinition.execute(flowContext);
 		} catch (Exception exception) {
-			throw new SystemException(exception.getMessage(), exception,
+			throw new SystemError(exception.getMessage(), exception,
 					PlatformStatusCode.ERROR_OCCURED_DURING_BUSINESS_PROCESSING);
 		}
 		LOGGER.debug("Async call completed {} ", "sendWelcomeNotification");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Async call completed {} " 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://ec2-52-23-152-124.compute-1.amazonaws.com:9000/coding_rules#rule_key=squid%3AS1192)